7_self_balancing/main/self_balancing.c

```diff
@@ -57,7 +59,15 @@ void calculate_motor_command(const float pitch_error, float *motor_cmd)
 
 	pitch_correction = P_term + I_term + D_term;
 
-    plot_graph(P_term, D_term, I_term, pitch_correction, pitch_error);
+	pg_data.p_term = P_term;
+	pg_data.d_term = D_term;
+	pg_data.i_term = I_term;
+	pg_data.pitch_corr = pitch_correction;
+	pg_data.pitch_err = pitch_error;
+	
+	plot_graph_data_t *pg_data_handle = &pg_data;
+
+	xQueueSend(plot_graph_queue, (void *) &pg_data_handle, (TickType_t) 0);
```

Comment:
Remove extra space and please follow good coding standards 
`xQueueSend(plot_graph_queue, (void *)&pg_data_handle, (TickType_t) 0);`

Reply:
Yes, Updated